deploy/src/erc_to_native/home.js

```diff
@@ -24,11 +27,15 @@ const {
   HOME_MIN_AMOUNT_PER_TX,
   HOME_REQUIRED_BLOCK_CONFIRMATIONS,
   FOREIGN_DAILY_LIMIT,
-  FOREIGN_MAX_AMOUNT_PER_TX
+  FOREIGN_MAX_AMOUNT_PER_TX,
+  REWARDABLE_VALIDATORS,
+  BRIDGE_FEE
 } = env
 
 const DEPLOYMENT_ACCOUNT_ADDRESS = privateKeyToAddress(DEPLOYMENT_ACCOUNT_PRIVATE_KEY)
 
+const useRewardableValidators = REWARDABLE_VALIDATORS === 'true'
```

Comment:
1. Consider to name the variable `isRewardableBridge`
2. let's use `false` by default

Reply:
1. Done https://github.com/poanetwork/poa-bridge-contracts/pull/134/commits/51a31be31f1e70380159b78a7ac6cd8b2b7eecdd
2. `HOME_REWARDABLE` and `FOREIGN_REWARDABLE` are set to `false` on `.env.example`